drivers/uktty/ns16550/ns16550.c

```diff
@@ -29,35 +29,54 @@
  */
 #include <libfdt.h>
 #include <uk/assert.h>
+#include <uk/boot/earlytab.h>
 #include <uk/config.h>
 #include <uk/init.h>
+#include <uk/ofw/fdt.h>
 #include <uk/plat/common/bootinfo.h>
 
+#if CONFIG_LIBUKLIBPARAM
```

Comment:
Those guards aren't needed because libuklibparam is providing the header always and switches configuration of the registration macros.

Reply:
I hadn't noticed it's always on, thanks for that 👍🏼